recipes/systemc/2.3.3/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SystemcConan(ConanFile):
+    name = "systemc"
```

Comment:
```suggestion
    name = "systemc_accellera"
```
Granted, accellera's library is the only open sources version, but there do exist other systemc libraries by other EDA's.

Reply:
True, but should I also rename the directory?

Reply:
The name of the directory should be the same as the name of the recipe.
But renaming the recipe is up to you.

Reply:
You can keep systemc in this case. As we only distribute packages for open source projects, and other package managers (VCPKG, ...) are using systemc only.